Doc/library/xmlrpc.server.rst

```diff
@@ -185,6 +192,37 @@ server::
    # Print list of available methods
    print(s.system.listMethods())
 
+Since version 3.7, :meth:`register_function` can also be used as a decorator. The
+previous server example can register functions in a decorator way::
+
+   from xmlrpc.server import SimpleXMLRPCServer
+   from xmlrpc.server import SimpleXMLRPCRequestHandler
```

Comment:
This is to be consistent with previous examples. I don't want to change this alone. :-)